components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,85 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@include exports("detail-page-sidebar") {
+  .bx--detail-page-sidebar__container {
+    margin: 0 5% 0 0;
+    .details-description,
+    .tag-description {
+      font-size: 0.875rem;
+      word-wrap: break-word;
+    }
+    .tag-description {
+      margin: 0 0 2rem 0;
+      > span {
+        font-weight: 600;
+      }
+    }
+    .description-label {
+      margin: 0;
+      overflow: visible;
+      color: $support-04;
+    }
+    .description-tag-docs-link {
+      display: block;
+    }
+  }
+
+  .bx--artifact-details-sidebar-container {
+    .description-label {
+      margin: 0 0 1rem 0;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+    .details-container {
+      margin: 2rem 0 2rem 0;
+      .inset-container {
+        margin: 0 0 .5rem 0;
+        line-height: 1.5;
+        font-size: .875rem;
+        .label {
+          text-transform: uppercase;
+          min-width: 30%;
+          margin: 0 .5rem 0 0;
+          display: inline-block;
+        }
+        .value {
+          white-space: nowrap;
+          text-overflow: ellipsis;
+          display: inline-block;
+          color: $support-04;
+        }
+      }
+    }
+  }
+
+  @include breakpoint(bp--sm--major) {
```

Comment:
This is a very recent change, but we're moving towards using `@media` query syntax and then using a function to pull in the value. Looks something like 

```scss
  @media (min-width: breakpoint('sm')) {
    ... content here
  }
```

We've found it more readable. Additionally, all media queries should be nested within the class they change. So it should be

```scss
.bx--detail-page-sidebar__container {
    ... mobile styles here

   @media min-width.. {
      small and above styles here
    }
}
```